plugins/graphql-kotlin-gradle-plugin/build.gradle.kts

```diff
@@ -9,6 +9,10 @@ val kotlinCoroutinesVersion: String by project
 val wireMockVersion: String by project
 val mustacheVersion: String by project
 
+repositories {
+    jcenter()
```

Comment:
its already defined in the root build file

Reply:
Removing..